src/custom/utils/analytics.ts

```diff
@@ -1,7 +1,7 @@
-import { isDev, isStaging, isProd, isEns, isPreStaging } from './environments'
+import { isLocal, isDev, isPr, isStaging, isProd, isEns, isBarn } from './environments'
 
 function getAnalyticsId(): string | undefined {
-  if (isDev || isPreStaging) {
+  if (isLocal || isDev || isPr || isBarn) {
```

Comment:
Maybe barn should use the same analytics as staging? Or even it's own.

Reply:
okok